SublimeLinter.sublime-settings

```diff
@@ -88,6 +88,12 @@
     // Highlight problems in the minimap.
     "show_marks_in_minimap": true,
 
+    // Show the number of errors and warnings in the current view
+    "statusbar.counters": true,
+
+    // Show the messages for problems at your cursor position
+    "statusbar.messages": true,
```

Comment:
Shouldn't these rather be more explicit and say "show_counters" and "show_messages"?

Reply:
I get this point for the variables, but showing stuff is all the statusbar does. Also, the comment is there.